Sources/Nitrozen-SwiftUI/Elements/Stepper/NitrozenStepperView.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  NitrozenStepperView.swift
+//  
+//
+//  Created by Rushang Prajapati on 24/04/23.
+//
+
+import Foundation
+import SwiftUI
+
+public struct NitrozenStepperView: View {
+    
+    @Binding var inputValue:String
+    var appearance: NitrozenAppearance.StepperView
+    var maxNumber:Int
```

Comment:
do we have option to limit the min number??  Always start with Apple provided native APIs when you start working on anything that is already implement.

Check below code how we can use iOS Provided stepper.
`struct StepperView: View {
    @State private var value = 0
    let step = 5
    let range = 1...50

    var body: some View {
        Stepper(value: $value,
                in: range,
                step: step) {
            Text("Current: \(value) in \(range.description) " +
                 "stepping by \(step)")
        }
            .padding(10)
    }
}`

Ref: https://developer.apple.com/documentation/swiftui/stepper

Reply:
Tip: always think about all the possible use cases and try to predict universal usage of any solution you are providing -- that will make you real Software Engineer.